Python/pystate.c

```diff
@@ -2009,42 +2016,57 @@ _Py_GetConfig(void)
 
 #define MINIMUM_OVERHEAD 1000
 
-PyObject **
-_PyThreadState_PushLocals(PyThreadState *tstate, int size)
+_Py_NO_INLINE PyObject **
```

Comment:
Why `_Py_NO_INLINE`?

Reply:
This is the slow path. We want to make sure the fast path is inlined.